src/MyEventEmitter.js

```diff
@@ -1,14 +1,58 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.events = {};
+  }
+  on(name, callback) {
+    if (!this.events[name]) {
+      this.events[name] = [];
+    }
+    this.events[name].push(callback);
+  }
+  once(name, callback) {
+    const wrapper = (...args) => {
+      callback(...args);
+      this.off(name, wrapper);
+    };
+
+    this.on(name, wrapper);
+  }
+  off(name, callback) {
+    if (this.events[name]) {
+      this.events[name] = this.events[name].filter((cb) => cb !== callback);
+    }
+  }
+  emit(name, ...args) {
+    if (this.events[name]) {
+      this.events[name].forEach((callback) => callback(...args));
+    }
+  }
+  prependListener(name, callback) {
+    if (!this.events[name]) {
+      this.events[name] = [];
+    }
+
+    this.events[name].unshift(callback);
+  }
+  prependOnceListener(name, callback) {
+    const wrapper = (...args) => {
+      callback(...args);
+      this.off(name, wrapper);
+    };
```

Comment:
Similar to the `once` method, ensure that the `off` method is correctly removing the wrapper function by reference in the `prependOnceListener` method.